Source/WebKit/Scripts/webkit/messages_unittest.py

```diff
@@ -55,6 +55,8 @@
     'TestWithStreamServerConnectionHandle',
     'TestWithEnabledBy',
     'TestWithEnabledIf',
+    'TestWithEnabledByAndConjunction',
```

Comment:
Are the two changes here related to the generate-serializers.py change? These don't seem to be related to the maySupportWebKitSecureEncoding parts?

Reply:
The patch updates the files generated by `make -C Source/WebKit/Scripts/webkit/tests`, as it has an effect on the generated content (e.g. new function is named `dictionaryForWebKitSecureCodingTypeFromWKKeyedCoder`). The auto generated files include message receivers for `TestWithEnabledByAndConjunction` and `TestWithEnabledByOrConjunction`, as there are corresponding messages.in files.
After this change, a unit test that checks the existing message receivers fails, because these two message receivers are not listed in expectation, so this change is updating the expectation so that the test can pass.
Technically the unit test should be fixed when the messages.in files were added; and it's discovered by my change.